### DISCUSSION TITLE
How to i build my project into desktop or apk with this file structure

### DISCUSSION THREAD

**Discussion by cupux** [flet-dev/flet#2754] [Q&A]
### Question

i have seperated my code into multiple sections in different files and folders but when i try to use flet park linux <app name> 
it only takes that particular app which is the main app. how do i build all together?

### Code sample

```python
import flet as ft

from router import views_handler

def main(page: ft.Page):
    
    def route_change(route):
        page.views.clear()
        page.views.append(
            views_handler(page)[page.route]
        )
        
        
    
    page.on_route_change = route_change
    page.go("/login")


ft.app(target=main, assets_dir="images")
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by lekshmanmj** [comment]
please try with ```flet build``` command.
for more details: https://flet.dev/docs/guides/python/packaging-app-for-distribution/
